c_glib/test/test-feather-file-reader.rb

```diff
@@ -18,26 +18,11 @@
 class TestFeatherFileReader < Test::Unit::TestCase
   include Helper::Buildable
 
-  def setup_file(data)
+  def setup_file(table)
     tempfile = Tempfile.open("arrow-feather-file-reader")
     output = Arrow::FileOutputStream.new(tempfile.path, false)
     begin
-      writer = Arrow::FeatherFileWriter.new(output)
-      begin
-        if data[:description]
-          writer.description = data[:description]
-        end
-        writer.n_rows = data[:n_rows] || 0
-        if data[:table]
-          writer.write(data[:table])
-        elsif data[:columns]
-          data[:columns].each do |name, array|
-            writer.append(name, array)
-          end
-        end
-      ensure
-        writer.close
-      end
+      Arrow::feather_write_file(table, output)
```

Comment:
This API is strange in GLib.
I'll improve this API.

Reply:
Thank you, I wasn't sure how to handle non-classes in GLib, didn't see too many examples

Reply:
Sorry about the Ruby failure, I am not set up locally to run the red-arrow tests but I will do that soon so I can handle these issues better in the future

Reply:
No problem. I'll also fix the Ruby failure.

Reply:
Done.
I'll improve `GArrowFeatherWriteProperties` in a follow-up task.